Makefile

```diff
@@ -179,6 +180,19 @@ api-docs: crd-ref-docs ## Creates API docs using https://github.com/elastic/crd-
 build: generate fmt vet ## Build manager binary.
 	go build -o bin/manager main.go
 
+# Build multi-arch image
+
+PLATFORMS ?= linux/amd64,linux/s390x,linux/ppc64le
+.PHONY: docker-buildx
+docker-buildx: ## Build and push docker image for the manager for cross-platform support
+	# copy existing Dockerfile and insert --platform=${BUILDPLATFORM} into Dockerfile.cross, and preserve the original Dockerfile
```

Comment:
Make Dockerfile.cross target and dependency of it.

Reply:
Since its generated on the fly & removed after execution , i don't understand the use of keeping it as target.